tasks/taxon_id/task_nextclade.wdl

```diff
@@ -168,76 +169,76 @@ task nextclade_output_parser {
-      File? reference_tree_json
-      File? qc_config_json
-      File? gene_annotations_gff
-      File? pcr_primers_csv
-      File? virus_properties
-      String docker = "us-docker.pkg.dev/general-theiagen/nextstrain/nextclade:2.14.0"
-      String dataset_name
-      String? dataset_reference
-      String? dataset_tag
-      Int disk_size = 50
-    }
-    String basename = basename(genome_fasta, ".fasta")
-    command <<<
-        NEXTCLADE_VERSION="$(nextclade --version)"
-        echo $NEXTCLADE_VERSION > NEXTCLADE_VERSION
-
-        nextclade dataset get \
-          --name="~{dataset_name}" \
-          ~{"--reference " + dataset_reference} \
-          ~{"--tag " + dataset_tag} \
-          -o nextclade_dataset_dir \
-          --verbose
-
-        # If no referece sequence is provided, use the reference tree from the dataset
-        if [ -z "~{reference_tree_json}" ]; then
-          echo "Default dataset reference tree JSON will be used"
-          cp nextclade_dataset_dir/tree.json reference_tree.json
-        else
-          echo "User reference tree JSON will be used"
-          cp ~{reference_tree_json} reference_tree.json
-        fi
-
-        tree_json="reference_tree.json"
-
-        set -e
-        nextclade run \
-            --input-dataset=nextclade_dataset_dir/ \
-            ~{"--input-root-seq " + root_sequence} \
-            --input-tree ${tree_json} \
-            ~{"--input-qc-config " + qc_config_json} \
-            ~{"--input-gene-map " + gene_annotations_gff} \
-            ~{"--input-pcr-primers " + pcr_primers_csv} \
-            ~{"--input-virus-properties " + virus_properties}  \
-            --output-json "~{basename}".nextclade.json \
-            --output-tsv  "~{basename}".nextclade.tsv \
-            --output-tree "~{basename}".nextclade.auspice.json \
-            --output-all=. \
-            "~{genome_fasta}"
-    >>>
-    runtime {
-      docker: "~{docker}"
-      memory: "8 GB"
-      cpu: 2
-      disks:  "local-disk " + disk_size + " SSD"
-      disk: disk_size + " GB" # TES
-      dx_instance_type: "mem1_ssd1_v2_x2"
-      maxRetries: 3
-    }
-    output {
-      String nextclade_version = read_string("NEXTCLADE_VERSION")
-      File nextclade_json = "~{basename}.nextclade.json"
-      File auspice_json = "~{basename}.nextclade.auspice.json"
-      File nextclade_tsv = "~{basename}.nextclade.tsv"
-      String nextclade_docker = docker
-      File netclade_ref_tree = "reference_tree.json"
-    }
+  meta {
+    description: "Nextclade task to add samples to either a user specified or a nextclade reference tree."
+  }
+  input {
+    File genome_fasta
+    File? root_sequence
+    File? reference_tree_json
+    File? qc_config_json
+    File? gene_annotations_gff
+    File? pcr_primers_csv
+    File? virus_properties
+    String docker = "us-docker.pkg.dev/general-theiagen/nextstrain/nextclade:2.14.0"
+    String dataset_name
+    String? dataset_reference
+    String? dataset_tag
```

Comment:
What happens if a dataset tag is not designated?

Reply:
If no organism is specified, then Nextclade will fail.